src/server/package.json

```diff
@@ -18,7 +18,8 @@
     "express": "^4.18.2",
     "express-async-handler": "^1.2.0",
     "express-session": "^1.17.3",
-    "mongoose": "^6.8.3"
+    "mongoose": "^6.8.3",
+    "multer": "^1.4.5-lts.1"
```

Comment:
Please install the `react-icons` package by running `npm install react-icons`, and make sure that it's on this file as a dependency. I faced an error without having it installed.

Reply:
Done!